inst/app.crunch.io/api/datasets/1/folders/1.json

```diff
@@ -0,0 +1,35 @@
+{
+    "self": "https://app.crunch.io/api/datasets/1/folders/1/",
+    "element": "shoji:catalog",
+    "size": 5,
+    "body": {
+        "name": "Group 1"
+    },
+    "index": {
+        "../3/": {
```

Comment:
I think this would be `./3/` ? Since its path would be `/folders/1/3/`? Given that the top level only had 1 and 2.

Reply:
No; all the folder entities are direct children of `folders/`, we're not exposing them as `folders/1/2/3/`.

Reply:
Ok, I can see how urls could get out of control

Reply:
@aminusfu What is the advantage of having all folders as direct children of root?

Reply:
* That the hierarchy is not exposed as part of the folder URL, this allows for different users to have different roots.
* Separation between URL and folder position, when a folder is moved, its URL doesn't change
* URLs don't get as big as the tree nesting

Reply:
Integrated that answer into https://github.com/Crunch-io/design/blob/folders-endpoint/features/datasets/variables/folders.md#folder-catalogs